examples/meta/src/multiclass/support_vector_machine.sg

```diff
@@ -4,16 +4,16 @@ CSVFile f_labels_train("../../data/classifier_4class_2d_linear_labels_train.dat"
 CSVFile f_labels_test("../../data/classifier_4class_2d_linear_labels_test.dat")
 
 #![create_features]
-RealFeatures features_train(f_feats_train)
-RealFeatures features_test(f_feats_test)
+Features features_train = features(f_feats_train)
+Features features_test = features(f_feats_test)
 MulticlassLabels labels_train(f_labels_train)
 MulticlassLabels labels_test(f_labels_test)
 #![create_features]
 
 #![set_parameters]
 real C = 1.0
 real epsilon = 0.0001
-GaussianKernel gauss_kernel(features_train, features_train, 15)
```

Comment:
kernel bandwidth changed from 15 to 1, need to regenerated integration testing data